plugins/modules/instance.py

```diff
@@ -523,6 +529,16 @@
         suboptions=linode_instance_placement_group_spec,
         description=["A Placement Group to create this Linode under."],
     ),
+    "lke_cluster_id": SpecField(
+        type=FieldType.integer,
+        description="The id of the LKE cluster associated with this Linode, if there is one.",
+    ),
+    "disk_encryption": SpecField(
+        type=FieldType.string,
+        description="The disk encryption status of this Linode. "
+        + "NOTE: Disk encryption may not currently be available to all users.",
+        choices=["enabled", "disabled", "pde"],
+    ),
```

Comment:
Looks like a redundant one

Reply:
It looks like disk_encryption should be able to be set either directly in the linode instance or within a disk that is attached to it. In order to allow this functionality, I had to add it to both the instance and instance_disk specs.